data/card_catalogue/greys.json

```diff
@@ -4,8 +4,11 @@
     "upgrade": "",
     "tier": "grey",
     "details": {
-      "rules": "Increase the value of a thrown d20 by 1",
-      "flavour": "The collective will of the people made manifest",
+      "rules": {
+        "active": "Increase the result of a roll by 1.",
+        "passive": ""
+      },
+      "flavour": "The collective will of the people made manifest.",
```

Comment:
'a roll' seems like the best way to solve the 'current or next roll' dilemma
also broadened the power of the card as we discussed

Reply:
It'll be interesting to see if there are any cases where that whiffs 🤔 I suppose that if you get this card randomly and you're forced to cast it you can always just do a random perception check or something.